openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx

```diff
@@ -52,12 +52,10 @@ import SVGIcons, { Icons } from './SvgUtils';
 
 export const getBadgeName = (tableType?: string) => {
   switch (tableType) {
-    case 'REGULAR':
-      return 'table';
     case 'QUERY':
-      return 'query';
+      return toLower(i18n.t('label.query'));
```

Comment:
For consistency, we should directly import `t` from the `i18next` and user here.

Reply:
sure!